std_msgs/Header header
std_msgs/Bool slow_stop
std_msgs/Bool fast_stop
float64 left_vel
float64 right_vel